docker/config/local/conseil/credentials.txt

```diff
@@ -0,0 +1,3 @@
+databaseName = "conseildatabase"
```

Comment:
How does this relate to the conseildb section of docker/config/local/conseil/conseil.conf?

Reply:
whatever you put there is what will replace that section in conseil.conf, if you put mikesdatabase, that's what conseil.conf will have once the container is built, it doesn't replace the original so we can have the ability to redeploy repeatedly